src/config/supported_distros.py

```diff
@@ -2,8 +2,9 @@
 'IBM z/OS ™': {
 	'z/OS Software': 'ZOS_Software_List.json'
 },
-'Debian 10': {
-	'Buster': 'Debian_Buster_List.json'
+'Debian': {
+	'Buster': 'Debian_Buster_List.json',
```

Comment:
Since these are code names and we want to make sure we're reflecting the official version number, maybe use '10 (Buster)' here? (and same for 11 (Bullseye))